src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -195,81 +195,86 @@ comment_id | Line/noticeAssignments/NoticeAssignment/NoticeRef | This field is p
 
 
 ## Reading of each folder
-In a **offre_** file, 2 **GeneralFrame** are expected in a **CompositeFrame/frames** node:
+In a **offre_*.xml** file, 2 **GeneralFrame** are expected in a **CompositeFrame/frames** node:
 * one with a `TypeOfFrameRef/@ref` containing the string `NETEX_STRUCTURE`
 * one with a `TypeOfFrameRef/@ref` containing the string `NETEX_HORAIRE`
 In the following chapters, the **CompositeFrame/frames/GeneralFrame/members** is ommitted.
 
 ### routes.txt
-`routes` are provided in each **offre_** file in the nodes **Route**. 
+`routes` are provided in each **offre_** file in the nodes **CompositeFrame/frames/GeneralFrame[]/members/Route[]**. The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-route_id | *Route/@id* | This field is prefixed. 
-route_name | *Route/Name* | 
-direction_type | *Route/DirectionType* | The value of this field is used without transformation.
+route_id | Route/@id | This field is prefixed. 
+line_id | Route/LineRef/@ref | This field is prefixed. 
+route_name | Route/Name | 
+direction_type | Route/DirectionType | The value of this field is used without transformation.
 destination_id |  | The `DirectionRef` of the Route doesn't link to a stop (neither stop_point nor stop_area), thus its value is not used.
-line_id | *Route/LineRef/@ref* | This field is prefixed. 
 
 **ServiceJourneyPattern references**
-All ServiceJourneyPattern of a `route` are stored as complementary `object_codes`. ServiceJourneyPattern nodes are listed in the same parent node as Route nodes.
+
+All ServiceJourneyPattern of a `route` are stored as complementary `object_codes`. ServiceJourneyPattern nodes are listed in the nodes `CompositeFrame/frames/GeneralFrame[]/members/ServiceJourneyPattern[]`. The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
 object_type |  | fixed value `route` 
-object_id | *ServiceJourneyPattern/RouteRef/@ref* | This field is prefixed. 
+object_id | ServiceJourneyPattern/RouteRef/@ref | This field is prefixed. 
 object_system |  | fixed value `Netex_ServiceJourneyPattern`
```

Comment:
`Netex_ServiceJourneyPattern` or `NetexIDFM_ServiceJourneyPattern`?

Reply:
Good question. I hope this code to be consistant in other netex profiles, and there is also a `Netex_PrivateCode` in Line object. Let's keep it like that.